test/geocoders/what3words.py

```diff
@@ -80,9 +80,9 @@ async def test_not_exactly_one(self):
         )
 
     async def test_result_language(self):
-        await self.geocode_run(
-            {"query": "piped.gains.jangle", "lang": 'DE'},
-            {"address": 'fortschrittliche.voll.schnitt'},
+        await self.reverse_run(
```

Comment:
Related to API v3 changes for language param requirement, I've modified the `test_result_language` test to use reverse function instead.